olive/workflows/run/config.py

```diff
@@ -10,7 +10,6 @@
 from olive.azureml.azureml_client import AzureMLClientConfig
 from olive.common.config_utils import ConfigBase, validate_config
 from olive.data.config import DataConfig
-from olive.data.constants import DEFAULT_HF_DATA_CONTAINER_NAME, DefaultDataContainer
 from olive.data.container.huggingface_container import HuggingfaceContainer
```

Comment:
We can remove `DEFAULT_HF_DATA_CONTAINER_NAME` from olive.data.constants as there is no case to use that.